app/models/user.rb

```diff
@@ -1,4 +1,5 @@
 class User < ApplicationRecord
+  has_many :microposts, dependent: :destroy
   attr_accessor :remember_token, :activation_token, :reset_token
```

Comment:
Thứ tự như này mới đúng chuẩn nha e
```suggestion
  attr_accessor :remember_token, :activation_token, :reset_token
  has_many :microposts, dependent: :destroy
```

Reply:
dạ vâng ạ